tests/run-pass/simd-intrinsic-generic-elements.rs

```diff
@@ -7,10 +7,6 @@ struct i32x2(i32, i32);
 #[repr(simd)]
 #[derive(Copy, Clone, Debug, PartialEq)]
 #[allow(non_camel_case_types)]
-struct i32x3(i32, i32, i32);
```

Comment:
we may need to reintroduce this (at least I was told in https://github.com/rust-lang/rust/issues/27731#issuecomment-775276211 that we need to support these again). So we should probably have an issue that reminds us to re-add this once it works again